### PR TITLE
Adds LCP guidance.

### PR DESCRIPTION
**Pull request by malchata**
Changes proposed in this pull request:

- Adds guidance around LCP on the [serve responsive images guide](https://web.dev/serve-responsive-images/).

### CODE REVIEW

File: src/site/content/en/fast/serve-responsive-images/index.md [hunk 1]
```diff
@@ -21,6 +21,24 @@ Serving desktop-sized images to mobile devices can use 2–4x more data than
...
+
+When you serve responsive images, you're evaluating the display capabilities of
+the user's device and choosing one of a set of image candidates that are optimal
+for display based on those criteria. The result&mdash;as stated previously&mdash;is
+rather than delivering too much image data to devices that won't benefit from it,
+you're serving an appropriately sized image for the device. For smaller devices such
+as phones and tablets, this equates to reduced data usage as compared to devices
+with larger screens, such as laptops.
+
+The effects of this on performance go beyond faster loading times for images. For
```
Comment (tunetheweb) [inline src/site/content/en/fast/serve-responsive-images/index.md:34]: Not sure about this first sentence. "It's more than just faster loading images. For example you can have a faster loading LCP image". That's just a faster loading image isn't it?

I do agree with the phrase, but think we should give a better example (while also leaving the important LCP example in place).

What about saying it can affect the load time of the rest of the page as large images can hold up all other resources due to bandwidth constraints? And it also impacts Core Web Vitals. For example if your LCP...

File: src/site/content/en/fast/serve-responsive-images/index.md [hunk 2]
```diff
@@ -21,6 +21,24 @@ Serving desktop-sized images to mobile devices can use 2–4x more data than
...
+example, if your page's [LCP candidate](/lcp/#what-elements-are-considered) is an image,
```
Comment (tunetheweb) [inline src/site/content/en/fast/serve-responsive-images/index.md:35]:
```suggestion
example, if your page's [LCP element](/lcp/#what-elements-are-considered) is an image,
```

File: src/site/content/en/fast/serve-responsive-images/index.md (hunk 1 above)
Comment (malchata) [reply src/site/content/en/fast/serve-responsive-images/index.md:34]: The point I guess I was (poorly) trying to make is that the effect extends to the LCP metric, which is a measure of perceptual performance. I'll have to workshop this.

Comment (malchata) [reply src/site/content/en/fast/serve-responsive-images/index.md]: I revised the language in this paragraph, @tunetheweb.

**Review by tunetheweb — APPROVED** [review]

File: src/site/content/en/fast/serve-responsive-images/index.md [hunk 3]
```diff
@@ -21,6 +21,26 @@ Serving desktop-sized images to mobile devices can use 2–4x more data than
...
+rather than delivering too much image data to devices that won't benefit from it,
+you're serving an appropriately sized image for the device. For smaller devices such
+as phones and tablets, this equates to reduced data usage as compared to devices
+with larger screens, such as laptops.
+
+The effects of faster image loading can also extend to your page's [Largest Contentful Paint (LCP)](/lcp/).
+For example, if your page's [LCP element](/lcp/#what-elements-are-considered) is an image,
+you're reducing that LCP candidate's [resource load time](/optimize-lcp/#3-reduce-resource-load-time).
+
+Lower resource load times will lower the load time for an LCP image, which will improve
```
Comment (philipwalton) [inline src/site/content/en/fast/serve-responsive-images/index.md:38]: This phrase ("Lower resource load times will lower the load time for an LCP image") feels redundant/repetitive to me. Also, it's not guarantee that a lower resource load time will improve LCP.

I'd rephrase this: "A lower LCP resource load time will generally result in a better overall LCP score."

**Review by philipwalton — APPROVED** [review]
LGTM (modulo one comment).
